<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitokuWindow</class>
 <widget class="QMainWindow" name="GitokuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>751</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="ui_repository_line_edit"/>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="ui_diff_viewer_tabwidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="ui_staged_dock">
   <property name="windowTitle">
    <string>Staged</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTableView" name="ui_staged_tableview"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ui_unstaged_dock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Unstaged</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTableView" name="ui_unstaged_tableview"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ui_workingtree_dock">
   <property name="windowTitle">
    <string>Working Tree</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="ui_workingtree_tableview"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ui_staged_tableview</tabstop>
  <tabstop>ui_unstaged_tableview</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ui_repository_line_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>GitokuWindow</receiver>
   <slot>on_repository_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_unstaged_tableview</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>GitokuWindow</receiver>
   <slot>on_display_diff(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_repository_changed()</slot>
  <slot>on_display_diff(QModelIndex)</slot>
 </slots>
</ui>
